src/OrchardCore/OrchardCore.Infrastructure/Documents/DocumentManager.cs

```diff
@@ -40,13 +38,19 @@ public class DocumentManager<TDocument> : IDocumentManager<TDocument> where TDoc
             }
         }
 
+        public Type DocumentStoreServiceType { get; set; }
```

Comment:
Could be readonly property?

Reply:
Just tried but we can't as there are derived types that assign it